src/ngx_http_lua_socket_tcp.c

```diff
@@ -5270,9 +5268,8 @@ ngx_http_lua_ssl_free_session(lua_State *L)
 
     psession = lua_touserdata(L, 1);
     if (psession && *psession != NULL) {
-        ngx_log_debug2(NGX_LOG_DEBUG_HTTP, ngx_cycle->log, 0,
-                       "lua ssl free session: %p:%d", *psession,
-                       (*psession)->references);
```

Comment:
There is no 1.1 equivalent. As the tlsext_status_expected field above, these are intended to be internal APIs and applications are not supposed to use them. I don't see OpenSSL adding new interfaces for this any time soon TBH.